src/components/base/translate/Translate.json

```diff
@@ -749,7 +749,7 @@
         "da": "Filmatisering af",
         "en": "Adaption of"
       },
-      "hasadaption" : {
+      "hasadaption": {
         "da": "Filmatisering",
```

Comment:
sådan en har jeg også lavet

Reply:
Den kørte bare prettier på save. Der var vist et mellemrum for meget